ADR/0049-vsa-support.md

```diff
@@ -0,0 +1,69 @@
+# ADR-0049: Verification Summary Attestations for Release Policies
+
+Date: 2025-07-02
+
+## Status
+
+Proposed
+
+## Context
+
+We are facing several challenges with our current approach to Enterprise Contract (EC) verification
+and SDL checks:
+
+1. **Release-time EC verification performance**: Running EC at release time requires checking a
+   large number of attestations, which takes too long. Scaling to handle large applications like
+   OpenShift is proving challenging.
+2. **Pipeline arrangement limitations**: Currently we have to run all SDL checks in the build
```

Comment:
I didn't see in https://github.com/konflux-ci/architecture/pull/229/files a requirement to use the suggested format. Is it a requirement for moving SDL checks to the integration tests?

Reply:
No. I raised a question wrt to what exactly is being attested in ADR 48, and how the scan results can be obtained. https://github.com/konflux-ci/architecture/pull/229#discussion_r2353555357